config/samples/dataplane_v1beta1_openstackdataplanerole.yaml

```diff
@@ -11,7 +11,51 @@ spec:
     - run-os
   nodeTemplate:
     networkConfig:
```

Comment:
Looking at the patch linked in this PR , I noticed that we're creating a file in runner pod and then using it. I noticed that empty file in the runner pod the other day. Why're doing all this rather than just converting the yaml blob to json and set  `edpm_network_config_override` ansibe var[1] that ansible would have taken care of.

[1] https://github.com/openstack-k8s-operators/edpm-ansible/blob/main/roles/edpm_network_config/tasks/os_net_config.yml#L26-L34